lib/compat/wordpress-6.1/persisted-preferences.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/**
+ * Server-side requirements for database persisted preferences.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Register the user meta for persisted preferences.
+ */
+function gutenberg_configure_persisted_preferences() {
+	$user_id = get_current_user_id();
+	if ( empty( $user_id ) ) {
+		return;
+	}
+
+	global $wpdb;
+
+	$meta_key = $wpdb->get_blog_prefix() . 'persisted_preferences';
```

Comment:
Yep, there was some feedback about it on the old PR - https://github.com/WordPress/gutenberg/pull/19177#issuecomment-567601834.

I'll add an inline comment so that it's clear to users.